packages/store-ui/src/Carousel/index.tsx

```diff
@@ -78,4 +78,10 @@ const Carousel: FC<Props> = ({
   )
 }
 
-export default Carousel
+export {
```

Comment:
Please roll back to `export default` so someone may `React.lazy` import this component